cmd/snap-seccomp/main.go

```diff
@@ -476,6 +476,10 @@ var seccompResolver = map[string]uint64{
 	"KCMP_IO":        C.KCMP_IO,
 	"KCMP_SYSVSEM":   C.KCMP_SYSVSEM,
 	"KCMP_EPOLL_TFD": C.KCMP_EPOLL_TFD,
+
+	// man 2 pipe
+	// The flag is just O_EXCL which is more easily defined.
+	"O_NOTIFICATION_PIPE": C.O_EXCL,
```

Comment:
can you define the constant for kernels that don't have it in the block at the beginning of the file?

Reply:
I tried but cgo didn't want to pick it up.

I had a plain #define 

Separately the headers are broken and I cannot include the right header when other include files are in this order. :/

Reply:
Fixed, I must have had a typo.